runelite-client/src/main/java/net/runelite/client/plugins/cooking/CookingPlugin.java

```diff
@@ -45,22 +46,30 @@
 @PluginDescriptor(
 	name = "Cooking",
 	description = "Show cooking statistics",
-	tags = {"overlay", "skilling", "cook"}
+	tags = {"cookingOverlay", "skilling", "cook"}
```

Comment:
`tags` are for when users search the config panel for settings. I don't think anyone will search for `cookingOverlay`.

Reply:
victim of find & replace, fixing.